moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -222,16 +224,18 @@ struct CollisionRequest
 
 namespace DistanceRequestTypes
 {
+/** \brief Type of distance request */
 enum DistanceRequestType
 {
   GLOBAL,   /// Find the global minimum
   SINGLE,   /// Find the global minimum for each pair
   LIMITED,  /// Find a limited(max_contacts_per_body) set of contacts for a given pair
-  ALL       /// Find all the contacts for a given pair
+  ALL,      /// Find all the contacts for a given pair
```

Comment:
?

Reply:
My problem is this 
![image](https://user-images.githubusercontent.com/19279526/59663651-c6ac5d80-91af-11e9-8d6e-99533927f6c3.png)
which I wanted to correct. I just rebuild the docu and will push new changes which solve this (as the added comma actually did not solve it).

Reply:
For future reference, adding `///<` also will fix this (https://stackoverflow.com/a/4429308).